theme_orchid/views/snippets/s_kickoff.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="configurator_s_kickoff" inherit_id="website.configurator_s_kickoff">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="oe_img_bg" remove="parallax s_parallax_is_fixed" separator=" "/>
+        <attribute name="style" add="background-image: url('/web/image/website.s_kickoff_default_image'); background-position: 100% 100%;" separator=";"/>
+    </xpath>
+
+    <xpath expr="//span[hasclass('s_parallax_bg')]" position="replace"/>
```

Comment:
Maybe we should add a comment to make it easier to understand that we want to remove this element, and what it is. What do you think?